os/kernel/init/os_start.c

```diff
@@ -594,6 +594,10 @@ void os_start(void)
 	/* Bring Up the System ****************************************************/
 	/* Create initial tasks and bring-up the system */
 
+#ifdef CONFIG_PM
+	/* We cannot enter low power state until boot complete */
+	pm_stay(0, 0);
```

Comment:
Is it possible to use some definition instead of 0?

Reply:
done